rcl/test/rcl/test_events.cpp

```diff
@@ -72,22 +74,23 @@ class CLASSNAME (TestEventFixture, RMW_IMPLEMENTATION) : public ::testing::Test
     ret = rcl_node_init(this->node_ptr, name, "", this->context_ptr, &node_options);
     ASSERT_EQ(ret, RCL_RET_OK) << rcl_get_error_string().str;
     ts = ROSIDL_GET_MSG_TYPE_SUPPORT(test_msgs, msg, Strings);
+
+    default_qos_profile.history = RMW_QOS_POLICY_HISTORY_KEEP_LAST;
```

Comment:
Moved the qos policies being used earlier into one `default_qos_profile` because both publisher and subscription tests were using the same qos policy values.